Formula/juicefs.rb

```diff
@@ -0,0 +1,19 @@
+class Juicefs < Formula
+  desc "Cloud-based, distributed POSIX file system built on top of Redis and S3"
+  homepage "https://juicefs.com"
+  url "https://github.com/juicedata/juicefs/archive/refs/tags/v1.0.3.tar.gz"
+  sha256 "2614424873065704d4ab63d3b4ecb831854b3f635709c56dc57ff1400e3cd962"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    bin.install "juicefs"
```

Comment:
essentially the same, but better `go build` visibility

Reply:
Hi @chenrui333, thank you for the suggestions! 🤝

It may not be appropriate to simply replace the compilation part with go build, it may be better to follow the project's default compilation method. What do you think?

Reply:
that also works, we can revisit this later.